sdk/src/lib.rs

```diff
@@ -224,10 +225,10 @@ macro_rules! saturating_add_assign {
     }};
 }
 
-#[macro_use]
-extern crate serde_derive;
 pub extern crate bs58;
 extern crate log as logger;
+#[cfg_attr(not(target_os = "solana"), macro_use)]
```

Comment:
What's this change about?

Reply:
After moving out poh_config, we were getting an unused macro warning that this fixes